### PR TITLE
Backport Release 7.1.0-beta (0x V2 Mesh)

### PR DESCRIPTION
**Pull request by fabioberger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Version](https://img.shields.io/badge/version-7.0.1--beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
+[![Version](https://img.shields.io/badge/version-7.1.0--beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
```
Comment (jalextowle) [inline README.md]: Will we always add the `beta` tag to V2 releases?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
+## v7.1.0-beta
+
+### Features ✅
+
+ - Reduce startup time for Mesh node by only waiting for a block to be processed if Mesh isn't already sync'ed up to the latest block. ([#622](https://github.com/0xProject/0x-mesh/pull/622))
```
Comment (jalextowle) [inline CHANGELOG.md]: nit: `sync'ed` => `synced`

File: README.md (hunk 1 above)
Comment (fabioberger) [reply README.md]: Yes.

File: CHANGELOG.md (hunk 2 above)
Comment (fabioberger) [reply CHANGELOG.md]: This release note was taken from `development` so I'm going to leave as is.

**Pull request merged**